.github/ct-lint.yaml

```diff
@@ -10,3 +10,5 @@ chart-repos:
 - k8s-at-home-libraries=https://library-charts.k8s-at-home.com
 - k8s-at-home=https://k8s-at-home.com/charts
 - jetstack=https://charts.jetstack.io
+additional-commands:
+- ./hack/check-releasenotes.sh {{ .Path }}
```

Comment:
This makes it so that the ct lint step also checks if the changelog annotation was updated.